package-lock.json

```diff
@@ -414,6 +445,27 @@
         "node": ">=4"
       }
     },
+    "node_modules/lec": {
+      "version": "1.0.1",
+      "resolved": "https://registry.npmjs.org/lec/-/lec-1.0.1.tgz",
+      "integrity": "sha512-3pIFvYBrdpfMK2cxP4xRBOrM56x1A1SykesXJ1I8/b6sUWvPYhM8p/1jIig0kRrmaKhW9c182GRgNOwwUh3i3A==",
+      "dependencies": {
+        "coffee-script": "^1.10.0",
```

Comment:
uff, jesus… I srsly don’t want to pull in coffeescript of all things for something as trivial as this.

Reply:
Unfortunately it doesn't seem like ncc has any options there, seems to be hard coded to windows line endings which is odd. There's an [open issue about line endings](https://github.com/vercel/ncc/issues/992) which doesn't look like it's going to be fixed.

Ahh yes, that's a bit heavy isn't it :( 

I'll see if there's something lighter, thanks for the consideration.

Reply:
linefix seems to do the trick and has zero dependencies.